doc/rbd/rbd-openstack.rst

```diff
@@ -336,23 +337,23 @@ Now on every compute nodes edit your Ceph configuration file::
         log file = /var/log/qemu/qemu-guest-$pid.log
         rbd concurrent management ops = 20
 
-Configure the permissions of these paths::
+Configure permissions of these directories::
```

Comment:
I think maybe this should be

"Configure the permissions of these directories"

or

"Configure permissions for these directories"

Let me know what you think.

Reply:
Proper use of prepositions for abstract ideas can be ambiguous at times.  I think it's okay as written, but I've added `for`.